docs-ref-conceptual/param-persist-tutorial.md

```diff
@@ -1,310 +0,0 @@
-   az config param-persist show
-   ```
-
-   ```output
-   {
-     "all": {
-       "location": "eastus2",
-       "resource_group_name": "RG1forTutorial",
-       "storage_account_name": "sa1fortutorial"
-     }
-   }
-   ```
-
-1. Create a persisted parameter without creating a new resource.
-
-   If you don't want to create a new Azure resource, `resource_group_name` and `location` parameters can be stored by using noncreate commands like `show` or `list`.   See [Azure CLI persisted parameters](./param-persist-howto.md) for a full list of supported parameters,   and the action needed to retain values.  This example also removes all parameter values by using the [az config param-persist delete](/cli/azure/config/param-persist#az_param_persist_delete) command.
-
-   ```azurecli
-   # Clear all persisted parameters for demonstration.
-   az config param-persist delete --all
-
-   # List all storage accounts which will create the `resource_group_name` stored parameter value.
-   az storage account show --resource-group RG1forTutorial --name sa1fortutorial
-
-   # See the new stored value created for resource group.  The storage account name is only stored with a 'create' command.
-   az config param-persist show
-   ```
-
-   ```output
-   {
-     "all": {
-       "resource_group_name": "RG1forTutorial"
-     }
-   }
-   ```
-
-## 4. Replace persisted parameters
-
-Replacing a stored parameter value is as simple as executing a command containing a different value.
-
-1. Create new persisted parameters.
-   ```azurecli
-   # Clear all persisted parameters for demonstration
-   az config param-persist delete --all
-
-   # Create a storage account placing "location", "resource_group_name", and "storage_account_name" into persisted parameters
-   az storage account create --name sa1fortutorial --resource-group RG1forTutorial --location eastus2
-
-   # See persisted parameters entries
-   az config param-persist show
-   ```
-
-   ```output
-   {
-     "all": {
-       "location": "eastus2",
-       "resource_group_name": "RG1forTutorial",
-       "storage_account_name": "sa1fortutorial"
-     }
-   }
-   ```
-
-1. Replace the newly stored values.
-
-   ```azurecli
-   # Create a second storage account while changing both the "storage_account_name" and "location" persisted parameters
-   az storage account create --name sa2fortutorial --location westeurope
-
-   # See new persisted parameters
-   az config param-persist show
-   ```
-
-   ```output
-   {
-     "all": {
-       "location": "westeurope",
-       "resource_group_name": "RG1forTutorial",
-       "storage_account_name": "sa2fortutorial"
-     }
-   }
-   ```
-
-   > [!NOTE]
-   >
-   > Even if persisted parameters are turned on, you don't have to use them.  You can still
-   > execute commands with all parameter values specified.  However, be aware that with persisted parameters
-   > turned on, _you will be creating new persisted parameters, or overwriting existing ones._
```

Comment:
Question: If a persisted parameter is already set, like the `resource_group_name`, how do I call a command _without_  `--resource-group`, instead of specifying one to override the persisted parameter?